index.html

```diff
@@ -3,43 +3,95 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Week 1: HTML Tables</title><!-- Latest compiled and minified CSS -->
+  <title>State reserves</title><!-- Latest compiled and minified CSS -->
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <style>
     .header {
       text-align: center;
     }
   </style>
 </head>
-<body>
+        <style>
```

Comment:
It's best to have all of the style declarations in the head, so we should move these inside the original style declaration that centers the header text.